zenoh-java/build.gradle.kts

```diff
@@ -80,19 +43,22 @@ kotlin {
             kotlinOptions.jvmTarget = "11"
         }
         testRuns["test"].executionTask.configure {
-            val zenohPaths = "../zenoh-jni/target/debug"
+            val zenohPaths = "../zenoh-jni/target/$buildMode"
             jvmArgs("-Djava.library.path=$zenohPaths")
         }
     }
-    androidTarget {
-        publishLibraryVariants("release")
+    if (androidEnabled) {
+        androidTarget {
+            publishLibraryVariants("release")
+        }
     }
 
     @Suppress("Unused")
     sourceSets {
         val commonMain by getting {
             dependencies {
                 implementation("commons-net:commons-net:3.9.0")
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.3")
```

Comment:
Please remove this, this is a subtle difference with Kotlin. Here in Java we don't use coroutines.

Reply:
Fixed